evil-colemak-basics.el

```diff
@@ -118,6 +128,14 @@ rotated; see evil-colemak-basics-rotate-t-f-j."
           "j" 'evil-snipe-j
           "J" 'evil-snipe-J))
        (t (user-error "Invalid evil-colemak-basics-char-jump-commands configuration"))))
+    (when (eq evil-colemak-basics-layout-mod 'mod-dh)
+      (evil-define-key '(motion normal visual) keymap
+        "m" 'evil-backward-char)
+      (evil-define-key 'normal keymap
+	"h" 'evil-set-marker)
```

Comment:
nit: there are still weird indents here and below (:bulb:  try `aggressive-indent-mode`)